src/main/java/io/numaproj/numaflow/function/FunctionServer.java

```diff
@@ -90,6 +91,16 @@ public FunctionServer registerMapTHandler(MapTHandler mapTHandler) {
         return this;
     }
 
+    /**
+     * registers the map stream handler to the server.
+     * @param mapStreamHandler handler to process the message and assign event time.
+     * @return returns a new Function gRPC server.
+     */
+    public FunctionServer registerMapStreamHandler(MapStreamHandler mapStreamHandler) {
```

Comment:
Could you add a unit test at FunctionServerTest to cover map stream? How was this PR tested?

Reply:
Had it in the local branch missed it to commit...

Reply:
thanks! My second question is about sdk test, do we plan to add it for Java?

Reply:
yes, going to update the e2e test with java and python.